prov/efa/src/rxr/rxr_ep.c

```diff
@@ -389,6 +389,7 @@ void rxr_tx_entry_init(struct rxr_ep *ep, struct rxr_tx_entry *tx_entry,
 		tx_op_flags &= ~FI_COMPLETION;
```

Comment:
this commit message has a typo: effcient -> efficient

Also I don't understand what:
"the size of these packet to be close" means, can you elaborate? Reading the code it looks like you mean something like, keep data packet sizes as similar as possible.

Reply:
Yes. I want to make the data size as similar as possible.

Reply:
Updated the comment and commit title.